sdk/eventhub/perf-tests/event-hubs-track-1/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@azure-tests/perf-event-hubs-track-1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@azure/core-amqp": "^3.0.0",
+    "@azure/event-hubs": "^2.1.4",
+    "@azure/test-utils-perfstress": "file:../../../test-utils/perfstress/azure-test-utils-perfstress-1.0.0.tgz",
+    "@types/node": "^8.0.0",
+    "@types/uuid": "^8.0.0",
+    "dotenv": "^8.2.0",
+    "moment": "^2.24.0",
+    "ts-node": "^8.3.0",
+    "tslib": "^2.0.0",
+    "typescript": "4.1.2",
+    "uuid": "^8.3.0"
+  },
+  "private": true,
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build": "tsc -p .",
+    "build:samples": "echo skipped",
+    "build:test": "echo skipped",
+    "check-format": "prettier --list-different --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm test-dist typings *.tgz *.log",
+    "format": "prettier --write --config ../../../../.prettierrc.json --ignore-path ../../../../.prettierignore \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "echo skipped",
+    "lint:fix": "eslint package.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json src test --ext .ts -f html -o event-hubs-perf-test-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "perf-test:node": "ts-node test/index.spec.ts",
+    "prebuild": "npm run clean",
+    "setup": "node ../../../../common/tools/perf-tests-track-1-setup.js",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "echo skipped",
+    "unit-test": "echo skipped",
+    "test:browser": "echo skipped",
+    "test:node": "echo skipped",
+    "test": "echo skipped"
+  },
+  "dependencies": {
+    "prettier": "^2.3.2"
+  }
```

Comment:
`prettier` being a runtime dependency surely sounds a mistake, no? could you please clean up the dependencies and group runtime and dev ones accordingly.